homeassistant/components/velbus/config_flow.py

```diff
@@ -21,7 +21,9 @@
 def velbus_entries(hass: HomeAssistant) -> set[str]:
     """Return connections for Velbus domain."""
     return {
-        entry.data[CONF_PORT] for entry in hass.config_entries.async_entries(DOMAIN)
+        entry.data[CONF_PORT]
```

Comment:
I suggest replacing the current approach with this helper:
https://github.com/home-assistant/core/blob/08ebb9f31a1ca56e1573ed20daecd426505f5201/homeassistant/config_entries.py#L1311-L1315